src/main/scala/org/caoilte/spray/routing/LogAccessRouting.scala

```diff
@@ -55,7 +56,27 @@ class RequestAccessLogger(ctx: RequestContext, accessLogger: AccessLogger,
       accessLogger.logAccess(request, response, timeStampCalculator(Unit))
       forwardMsgAndStop(response)
     }
+    case Confirmed(ChunkedResponseStart(response), sentAck) => {
```

Comment:
is it safe to assume these will always be Confirmed?

ie sent via responder ! MessageChunk.withAck(sentAck)?